### ISSUE TITLE
Not found: Op type not registered 'BatchMatMulV2'

### ISSUE THREAD

**Issue by superhg2012** [tensorflow/tensorflow#34198]
**System information**
- Have I written custom code (as opposed to using a stock example script provided in TensorFlow): no
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04):  Linux Ubuntu 16.04
- TensorFlow version (use command below): 1.13.1  (docker build inside)
- Docker  tensorrtserver:19.06-py3
- GCC/Compiler version (if compiling from source):
- CUDA/cuDNN version:  CUDA Version 10.1.243 / 7.6.3
- GPU model and memory: T4 16G

You can collect some of this information using our environment capture
[script](https://github.com/tensorflow/tensorflow/tree/master/tools/tf_env_collect.sh)
You can also obtain the TensorFlow version with: 1. TF 1.0: `python -c "import
tensorflow as tf; print(tf.GIT_VERSION, tf.VERSION)"` 2. TF 2.0: `python -c
"import tensorflow as tf; print(tf.version.GIT_VERSION, tf.version.VERSION)"`

**Describe the current behavior**

I am using tensorrt-inference-server to deploy my tensorflow savedmodel files( tf version : 1.11.1).

the config.pbtxt 

`name: "jingjing"
platform: "tensorflow_savedmodel"
max_batch_size: 0
input [
  {
    name: "inputs"
    data_type: TYPE_INT32
    dims: [ -1]
  },
  {
    name: "input_lengths"
    data_type: TYPE_INT32
    dims: [ -1 ]
  },
  {
    name: "split_infos"
    data_type: TYPE_INT32
    dims: [ 1, -1 ]
  }
]
output [
  {
    name: "linear_wav_outputs"
    data_type: TYPE_FP32
    dims: [ -1 ]
  }
]
version_policy: { all { }}`
when trying to run the tensorrt inference server , errors will occur like below :
`failed to load 'jingjing' version 1: Not found: Op type not registered 'BatchMatMulV2' in binary running on fa60ca095bbf. Make sure the Op and Kernel are registered in the binary running in this process. Note that if you are loading a saved graph which used ops from tf.contrib, accessing (e.g.) `tf.contrib.resampler` should be done before importing the graph, as contrib ops are lazily registered when the module is first accessed.`

and when curl 
``id: "inference:0"
version: "1.3.0"
uptime_ns: 16498837986
model_status {
  key: "jingjing"
  value {
    config {
      name: "jingjing"
      platform: "tensorflow_savedmodel"
      version_policy {
        all {
        }
      }
      max_batch_size: 1
      input {
        name: "inputs"
        data_type: TYPE_INT32
        dims: -1
        dims: -1
      }
      input {
        name: "input_lengths"
        data_type: TYPE_INT32
        dims: -1
      }
      input {
        name: "split_infos"
        data_type: TYPE_INT32
        dims: 1
        dims: -1
      }
      output {
        name: "linear_wav_outputs"
        data_type: TYPE_FP32
        dims: -1
      }
      instance_group {
        name: "jingjing"
        count: 1
        gpus: 0
        kind: KIND_GPU
      }
      default_model_filename: "model.savedmodel"
    }
    version_status {
      key: 1
      value {
        ready_state: MODEL_UNAVAILABLE
      }
    }
    version_status {
      key: 2
      value {
        ready_state: MODEL_UNAVAILABLE
      }
    }
    version_status {
      key: 3
      value {
        ready_state: MODEL_UNAVAILABLE
      }
    }
    version_status {
      key: 4
      value {
        ready_state: MODEL_UNAVAILABLE
      }
    }
  }
}
ready_state: SERVER_READY``

**Describe the expected behavior**

the ready_state of model should be MODEL_READY

**Code to reproduce the issue**
`docker run --rm --gpus "device=3" --shm-size=1g --ulimit memlock=-1 \
--ulimit stack=67108864 -p8000:8000 -p8001:8001 -p8002:8002 \
-v/home/yichao.li/models:/models nvcr.io/nvidia/tensorrtserver:19.06-py3 \
trtserver --model-store=/models &`

please help to solve the " Op type not registered 'BatchMatMulV2'" issue

**Comment by oanush** [comment]
@superhg2012 ,
Can you share a standalone code to reproduce the error reported here?Thanks!

**Comment by superhg2012** [comment]
@oanush There is just one savedmodel.pb and variables files. I deployed it to tensorrtinference server container. then the error occurs.

**Comment by gowthamkpr** [comment]
@superhg2012 This is duplicate of this issue that you have created [here](https://github.com/NVIDIA/tensorrt-inference-server/issues/855).  I am afraid you have to do as mentioned in the comment [here](https://github.com/NVIDIA/tensorrt-inference-server/issues/855#issuecomment-553210106). Thats the only fix as of now.

Also please note that this is not a tensorflow issue.  Thanks!

**Comment by gowthamkpr** [comment]
Closing this issue as its not cause due to native tensorflow. Please add additional comments and we can open this issue again. Thanks!
